bucket/apache-tinkerpop-gremlin-console.json

```
@@ -6,7 +6,6 @@
     "url": "https://www.apache.org/dist/tinkerpop/3.6.0/apache-tinkerpop-gremlin-console-3.6.0-bin.zip",
     "hash": "sha512:727061ee2874087bc889cd26aa651537c96eaa84d9eaed568902de67d0765a9acca4cb6efa11786a45e6fdde5242df908ab2d84ea925dd87ae907537949b108a",
     "extract_dir": "apache-tinkerpop-gremlin-console-3.6.0",
```

You don't have to remove **bin**. There is a better way to solve this.

```suggestion
    "extract_dir": "apache-tinkerpop-gremlin-console-3.6.0",
    "pre_install": [
        "$cont = @(",
        "    '@echo off'",
        "    'cd /d \"%~dp0\"'",
        "    'gremlin.bat'",
        ")",
        "Set-Content \"$dir\\bin\\gremlin-scoop.bat\" $cont -Encoding ascii"
    ],
    "bin": [
        [
            "bin\\gremlin-scoop.bat",
            "gremlin"
        ]
    ],
```